.github/LICENSE.md

```diff
@@ -1,4 +1,4 @@
-MIT License
+### MIT License
```

Comment:
Hmm, I don't see why we should add text formatting to the license.

Reply:
I just thought it would be a nice touch lol, I'll change it back.